src/modules/commentPopup.js

```diff
@@ -0,0 +1,130 @@
+import { makeComment } from './apis.js';
+
+// get comment count from local storage
+const count = JSON.parse(localStorage.getItem('commentCounter'));
+
+// count the comments💬💬 of each rocket🚀
+const commentCount = async () => JSON.parse(localStorage.getItem('commentCounter'));
+
+// function render data of the homepage
+const popupView = async (
+  rocketId,
+  rocketImage,
+  rocketName,
+  successRate,
+  costPerLaunch,
+  weightMass,
+  rocketType,
+  commentData,
+) => {
+  // Retrieve comment count asynchronously
+  const commentCountValue = await commentCount();
+
+  const popupsContainer = document.querySelector('.popups-container');
+  let popup = '';
+  popup += `
+    <section class="each-popup">
```

Comment:
- [ ] When I try to add a comment, the request is sent to the API, and successfully created. However, the screen nor the counter gets updated. According to the requirements:

![image](https://github.com/til2to/javascript-capstone/assets/77069011/bd6daa49-7daa-4c1f-bb24-a5edc3999194)

[Screencast from 27-05-23 10:25:59.webm](https://github.com/til2to/javascript-capstone/assets/77069011/219f4c62-a0b2-4911-b98f-f26ae9776ca6)

Please consider fixing this! 🤓

Reply:
Alright. Thanks

Reply:
resolved